app/src/main/java/org/mozilla/focus/ext/Fragment.kt

```diff
@@ -12,5 +12,5 @@ import org.mozilla.focus.Components
 /**
  * Get the components of this application.
  */
-val Fragment.components: Components
-    get() = context!!.components
+val Fragment.components: Components?
```

Comment:
nit: I still don't love that it's unclear why `Fragment.components` can be null - that it's tied to the Context. I think I might actually prefer it to be `val Context.components: Components` (i.e. non-optional) to make it clear that it's the context we need to look out for.

Reply:
Okay, I removed this extension completely and access `components` via a `Context` in the fragments now.